packages/web-components/src/components.d.ts

```diff
@@ -12133,6 +12133,10 @@ export namespace Components {
           * Opens the pop up and returns true.
          */
         "show": () => Promise<true>;
+        /**
+          * When provided, will close the pop-up when a single selection is made.
+         */
+        "singleSelect": boolean;
```

Comment:
what about close-on-select instead? looks good to me though! will this fire a close event too?

Reply:
Yup, the `open` prop watch is handling the open/close events so this will emit a close event too. I also renamed it to be close-on-select